### ISSUE TITLE
[BUG] chrome://extensions/ page empty after syncing

### ISSUE THREAD

**Issue by not-lucky** [Alex313031/Thorium-Win#12]
Can't reproduce anymore but I saw the same issue mentioned here-

#https://github.com/Alex313031/Thorium/issues/47

So I dont know what solved this exactly but the issue was still there even after uninstalling and installing Thorium (both this version and AVX2 version), but I can tell some of the steps I did from start to finish.

TLDR is at the end.

So I installed Thorium yesterday for the first time and installed ublock origin, and tested the browser a bit. After I was satisfied I logged in to sync. I didn't have any extensions data in [the chrome sync data on their site](https://chrome.google.com/sync?hl=en-GB).

I was using ungoogled-chromium before this and I was wondering if I can just copy paste my extensions from there since I have more than 25 extension there.

So then I went to `C:\Users\lucky\AppData\Local\Thorium\User Data` and I couldn't find my profile under `Default` like in ungoogled-chromium, but found it in under the name `Profile1` instead. I copy pasted extensions but they were corrupted so I just manually removed them.

After that I manually installed all of them.

Then I wanted to disable some extensions but I noticed the chrome://extensions/ page was empty, but I can still see the extensions in the top bar and all of them were still present in the the extension folder.

So I uninstalled and installed and tried AVX2 version too but after syncing, the extensions page was empty.

Then I went to github repo to see if this was mentioned and it was mentioned in the main Thorium repo, and wasn't reproducible by some people.

So I enabled logging like this `C:\Users\lucky\AppData\Local\Thorium\Application\thorium.exe --enable-logging --v=1` and opened from terminal.

This was my log:

```
[10256:12224:0806/081609.642:ERROR:device_event_log_impl.cc(214)] [08:16:09.643] USB: usb_device_handle_win.cc:1048 Failed to read descriptor from node connection: A device attached to the system is not functioning. (0x1F)
[10256:12224:0806/081609.652:ERROR:device_event_log_impl.cc(214)] [08:16:09.653] USB: usb_device_handle_win.cc:1048 Failed to read descriptor from node connection: A device attached to the system is not functioning. (0x1F)
[10256:12224:0806/081609.652:ERROR:device_event_log_impl.cc(214)] [08:16:09.653] USB: usb_device_handle_win.cc:1048 Failed to read descriptor from node connection: A device attached to the system is not functioning. (0x1F)
[10256:12224:0806/081617.958:ERROR:CONSOLE(167)] "Uncaught (in promise) Error: Assertion failed: Don't send themes to the chrome://extensions page", source: chrome://extensions/extensions.js (167)
```

Then I created a new profile and enabled some extensions in the new profile and checked out extensions page but it was good.

Then I quit the browser and opened again. It asked me to select the profile there and I deleted the second profile I created and closed the browser again.

After I opened the browser and checked the extensions page, it was back to normal and the log was normal too.

After this I went to my user profile, but it was no longer under `Profile1` but under `Default` this time.

I uninstalled and reinstalled Throrium after that and the extensions page was perfectly fine even after syncing, and my user data was always under `Default` now.

TLDR:

Extension page was bugged when my user data after syncing was under `Profile1` but after I created a new profile and deleted new profile, my user data was under `Default` and everything was fine afterwards

**Comment by Alex313031** [comment]
@not-lucky That helps alot! I think it is my patch to show all extensions/apps. I have it also set to show themes, but they dont show anyway, and i think this is causing the error. I will remove that from the patch since it wont show themes anyway.

**Comment by Alex313031** [comment]
@not-lucky Yep fixed it, thank you!! I will include you in the copyright for helping.

See > https://github.com/Alex313031/Thorium/commit/8d99f75ec4095425ae0eaa414354865fdab80066
